decode_test.go

```diff
@@ -991,6 +991,28 @@ func TestDecodeDefaultValues(t *testing.T) {
 	}
 }
 
+func TestDecodePartialKeys(t *testing.T) {
+	type defaultValueStruct struct {
+		Foo string `csv:"foo,partial"`
+		Bar int    `csv:"bar,partial"`
+	}
+	b := bytes.NewBufferString(`foolo,ambar
```

Comment:
isn't it wrong? should it not be:
```csv
foo,bar
x,42```

Reply:
No, that's the purpose of this feature, to match a substring of the header. (Maybe these names are a little bit stupid, I can change them to something more realistic)

Reply:
Oh ok I misunderstood, thanks for the clarification 🙂